utils/scarb-test-support/src/simple_http_server.rs

```diff
@@ -45,14 +46,18 @@ pub struct HttpLog {
 }
 
 impl SimpleHttpServer {
-    pub fn serve(dir: PathBuf) -> Self {
+    pub fn serve(dir: PathBuf, post_status: Option<u16>) -> Self {
```

Comment:
nitpick, I think that instead of passing and requiring this argument in a constructor I'd rather keep it as an `AtomicU16` as struct field and expose following method to use only when needed: `fn set_post_status(&mut self, status: u16)`

Reply:
This would have to be modified in router probably, that is defined in serve function. Not sure this will be this trivial, but if you insist I can try and research this.

Reply:
not insisting, do as you wish. just finding amount of stuff you're changing alerting